ToolkitApp/tabs/src/components/sample/Welcome.css

```diff
@@ -76,4 +76,88 @@ code {
   width:20px;
   height:auto;
   display:inline-block;
+}
+
+* {
```

Comment:
`*` is not recommend because this it also affect inner html elements. if this is a basic style, try to find appropriate element, or use `body`

Reply:
The CSS is quite messy, I'll be cleaning it up in a separate PR! I will note this down!